jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/PluginConfigurationProcessor.java

```diff
@@ -59,6 +68,35 @@ static AbsoluteUnixPath getAppRootChecked(JibPluginConfiguration jibPluginConfig
     }
   }
 
+  /**
+   * Gets the value of the {@code <to><image>} parameter. If this parameter is null,
+   * "gcr.io/distroless/java/jetty" is returned for war packaging, "gcr.io/distroless/java" is
+   * returned otherwise.
+   *
+   * @param jibPluginConfiguration the Jib plugin configuration
+   * @return the base image value
+   */
+  static String getBaseImage(JibPluginConfiguration jibPluginConfiguration) {
+    String baseImage = jibPluginConfiguration.getBaseImage();
+    if (baseImage == null) {
+      baseImage =
+          isWarPackaging(jibPluginConfiguration)
+              ? "gcr.io/distroless/java/jetty"
+              : "gcr.io/distroless/java";
+    }
+    return baseImage;
+  }
+
+  /**
+   * Returns true if the maven packaging type is "war"
+   *
+   * @param jibPluginConfiguration the Jib plugin configuration
+   * @return true if the maven packaging type is "war"
+   */
+  private static boolean isWarPackaging(JibPluginConfiguration jibPluginConfiguration) {
```

Comment:
style nit: in our codebase, `private static` methods go below (package-private) `static` methods.

Reply:
Done